integration/anchor/tests/call_anchor.spec.ts

```diff
@@ -19,13 +19,12 @@ async function newAccountWithLamports(connection: Connection): Promise<Keypair>
 }
 
 describe('Call Anchor program from Solidity via IDL', () => {
-
     it('call_anchor', async function () {
         // This program instantiates an anchor program, calls various functions on it and checks the return values
 
         const connection = new Connection("http://localhost:8899", {
             commitment: "confirmed",
-            confirmTransactionInitialTimeout: 100000,
+            confirmTransactionInitialTimeout: 1000000,
```

Comment:
In the `newConnection` helper in `solana/setup.ts`, this value is 10x. Is this intentional? Also can't we use the same helper here (making this timeout configurable if needed)?

Reply:
Good catch, they are different values, I've replaced them both with `1e6`.

There is no shared typescript code with between the (bare) solana tests and the anchor tests. :cry: